packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5310,14 +5330,15 @@ export default class Meeting extends StatelessWebexPlugin {
         this.reconnectionManager.resetReconnectionTimer();
 
         this.reconnect({networkDisconnect: true});
-        Metrics.postEvent({
-          event: eventType.ICE_END,
-          meeting: this,
-          data: {
-            canProceed: false,
-            errors: [
-              Metrics.generateErrorPayload(2004, false, MetricsError.name.MEDIA_ENGINE, undefined),
-            ],
+        NewMetrics.submitClientEvent({
+          name: 'client.ice.end',
+          payload: {canProceed: false},
+          options: {
+            meetingId: this.id,
+            error: {
+              // TODO: adapt this code to work in separate JIRA
```

Comment:
does the jira exist? can you create it so that it doesn't get forgotten?

Reply:
yes, https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-435138